src/api/app/controllers/webui/session_controller.rb

```diff
@@ -8,36 +8,46 @@ def new; end
   def create
     user = User.find_with_credentials(params[:username], params[:password])
 
-    if user && !user.is_active?
-      redirect_to(root_path, error: 'Your account is disabled. Please contact the administrator for details.')
+    if user.nil?
+      redirect_to(session_new_path, error: 'Authentication failed')
+
```

Comment:
why this blank line? blank lines are supposed to be used to separate different conceptual parts of the cope, and this return is related. Please remove this line. :wink: